CONTRIBUTING.md

```diff
@@ -103,15 +103,23 @@ pnpm test Alert
 
 Note that this bypasses the turborepo pipeline, so you will need to manually run any dependent tasks first. For example, if the `build` command of `react-workshop` relies on the `build` command of `@itwin/itwinui-react`, then you will need to manually run the `build` commands in the right order.
 ```
-pnpm --filter @itwin/itwinui-react build
-pnpm --filter react-workshop build
+pnpm --filter=@itwin/itwinui-react build
+pnpm --filter=react-workshop build
 ```
 
 This is why it's recommended to use the turbo `--filter` syntax whenever possible.
 ```
 pnpm run build --filter=react-workshop
 ```
 
+> [!NOTE]
+>
+> The `--filter` argument is used in both `turbo` and `pnpm`. To use it with:
+> * `turbo`: `pnpm run [command] --filter=[workspace]`
+>   * e.g. `pnpm run build --filter=react-workshop`
+> * `pnpm`: `pnpm --filter=[workspace] run [command]`
+>   * e.g. `pnpm --filter=react-workshop run build`
```

Comment:
> This looks good, but to make it even better we could add a short note (under "Running bespoke commands") to disambiguate between Pnpm's filter syntax and Turbo's filter syntax.

@mayank99 Is this like what you suggested?

Reply:
Can add links and explain the difference.

```suggestion
> [!NOTE]
>
> The `--filter` syntax is available in both `turbo` and `pnpm`. The usage looks slightly different:
> - `turbo`: `pnpm run [command] --filter=[workspace]`
>   - e.g. `pnpm run build --filter=@itwin/itwinui-react`
>   - This will also run any dependent tasks defined in the Turbo pipeline, but does not allow args. (See [Turborepo docs](https://turbo.build/repo/docs/core-concepts/monorepos/filtering)).
> - `pnpm`: `pnpm --filter=[workspace] run [command] [args]`
>   - e.g. `pnpm --filter=@itwin/itwinui-react run test Alert`
>   - This will only run the task per-workspace and supports additional args. (See [Pnpm docs](https://pnpm.io/filtering)).
```